lib/spotlight/dor/indexer.rb

```diff
@@ -150,9 +183,10 @@ def add_document_subtype(sdb, solr_doc)
         solr_doc['doc_subtype_ssi'] = subtype.first unless subtype.empty?
       end
 
+      # @note upcasing each word not the same as .captialize (lowercases the rest of the string), or .titleize (breaks CamelCase words apart)
       def add_donor_tags(sdb, solr_doc)
         donor_tags = sdb.smods_rec.note.select { |n| n.displayLabel == 'Donor tags' }.map(&:content)
-        insert_field solr_doc, 'donor_tags', upcase_first_character(donor_tags), :symbol # this is a _ssim field
+        insert_field solr_doc, 'donor_tags', donor_tags.map { |v| v.sub(/^./, &:upcase) }, :symbol # this is a _ssim field
```

Comment:
This seems less readable to me; were you trying to address the limited utility of the `upcase_first_character` method?  Another approach might be to extract it to its own class-level helper.